maxim/logger/openai/async_responses.py

```
@@ -143,17 +144,30 @@ def _start_trace_and_generation(
         trace: Optional[Trace] = None
         generation: Optional[Generation] = None
         try:
             trace = self._logger.trace({"id": final_trace_id, "session_id": session_id})
+            if trace_tags is not None and not isinstance(trace_tags, str):
+                scribe().warning(f"[MaximSDK][MaximAsyncOpenAIResponses] Trace tags must be a dictionary, got {type(trace_tags)}")
+            if trace_tags is not None and isinstance(trace_tags, str):
+                try:
+                    trace_tags = json.loads(trace_tags)
+                    for key, value in trace_tags.items():
+                        trace.add_tag(key, str(value))
+                except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)